source/dmocks/util.d

```diff
@@ -60,6 +63,17 @@ public class ExpectationViolationError : Error
     }
 }
 
+/**
+ * Thrown when an expectation violation was found during Fixture validation.
```

Comment:
What's "Fixture validation"? Maybe "final verification"?
(`dmocks` does not know what a fixture is.)

Reply:
fixed